api/host.py

```diff
@@ -90,7 +90,7 @@ def add_host_list(body):
                         logger.info("Tags from an HTTP request were ignored")
 
                     input_host = deserialize_host_http(host)
-                    (output_host, add_result) = _add_host(input_host)
+                    (output_host, _, add_result) = _add_host(input_host)
                     status_code = _convert_host_results_to_http_status(add_result)
                     response_host_list.append({"status": status_code, "host": output_host})
                     payload_tracker_processing_ctx.inventory_id = output_host["id"]
```

Comment:
This too extracts information from the serialized payload. If we retrieve the Host ID above, we can use it here.

```suggestion
                    payload_tracker_processing_ctx.inventory_id = host_id
```